_posts/2021-05-06-broker-tuning.md

```diff
@@ -398,7 +402,11 @@ Lag time puts an upper limit on the time to replicate a message to all in-sync r
 
 >Followers operate only to replicate messages from the partition leader and allow recovery should the leader fail. Followers do not normally serve clients, though [rack configuration](https://strimzi.io/docs/operators/latest/using.html#type-Rack-reference) allows a consumer to consume messages from the closest replica when a Kafka cluster spans multiple datacenters.
 
-A failed leader affects the balance of a Kafka cluster, as does the assignment of partition replicas to brokers. Leader rebalancing ensures that leaders are evenly distributed across brokers and brokers are not overloaded. You can control the frequency, in seconds, of leader rebalance checks and the maximum percentage of imbalance allowed for a broker before a rebalance is triggered.
+You can [use Cruise Control for Strimzi](https://strimzi.io/docs/operators/latest/using.html#cruise-control-concepts-str) to figure out replica assignments to brokers that balance load evenly across the cluster. It's calculation takes into account the differing load experienced by leaders and followers. A failed leader affects the balance of a Kafka cluster because the remaining brokers get the extra work of leading additional partitions.
```

Comment:
```suggestion
You can [use Cruise Control for Strimzi](https://strimzi.io/docs/operators/latest/using.html#cruise-control-concepts-str) to figure out replica assignments to brokers that balance load evenly across the cluster. Its calculation takes into account the differing load experienced by leaders and followers. A failed leader affects the balance of a Kafka cluster because the remaining brokers get the extra work of leading additional partitions.
```